bed/methods.go

```diff
@@ -57,10 +71,14 @@ func (b BedSlice) Write(file string) {
 	exception.PanicOnErr(err)
 }
 
+// WriteToFileHandle will write a bed struct to the
+// io.Writer
 func (b Bed) WriteToFileHandle(file io.Writer) {
 	WriteToFileHandle(file, b) //adaptive field writing seems most flexible for the method
 }
 
+// NextRealRecord keeps track of if we are done reading through
```

Comment:
This comment is true, but we should also add that it sets b equal to the next bed entry in the file (along with keeping track of if we are done reading the file).